scripts/o3de/o3de/gem_properties.py

```diff
@@ -88,26 +88,17 @@ def edit_gem_props(gem_path: pathlib.Path = None,
                          f' characters, and start with a letter.  {new_name}')
             return 1
         update_key_dict['gem_name'] = new_name
-    if new_display is not None:
-        update_key_dict['display_name'] = new_display
-    if new_origin is not None:
-        update_key_dict['origin'] = new_origin
-    if new_type is not None:
-        update_key_dict['type'] = new_type
-    if new_summary is not None:
-        update_key_dict['summary'] = new_summary
-    if new_icon is not None:
-        update_key_dict['icon_path'] = new_icon
-    if new_requirements is not None:
-        update_key_dict['requirements'] = new_requirements
-    if new_documentation_url is not None:
-        update_key_dict['documentation_url'] = new_documentation_url
-    if new_license is not None:
-        update_key_dict['license'] = new_license
-    if new_license_url is not None:
-        update_key_dict['license_url'] = new_license_url
-    if new_repo_uri is not None:
-        update_key_dict['repo_uri'] = new_repo_uri
+    
+    update_key_dict['display_name'] = new_display if isinstance(new_display, str) else ""
+    update_key_dict['origin'] = new_origin if isinstance(new_origin, str) else ""
+    update_key_dict['type'] = new_type if isinstance(new_type, str) else ""
+    update_key_dict['summary'] = new_summary if isinstance(new_summary, str) else ""
+    update_key_dict['icon_path'] = new_icon if isinstance(new_icon, str) else ""
+    update_key_dict['requirements'] = new_requirements if isinstance(new_requirements, str) else ""
+    update_key_dict['documentation_url'] = new_documentation_url if isinstance(new_documentation_url, str) else ""
+    update_key_dict['license'] = new_license if isinstance(new_license, str) else ""
+    update_key_dict['license_url'] = new_license_url if isinstance(new_license_url, str) else ""
+    update_key_dict['repo_uri'] = new_repo_uri if isinstance(new_repo_uri, str) else ""
```

Comment:
would you get the same behavior and simpler code without checking if it's a string? i.e.
```
update_key_dict['display_name'] = new_display if new_display else ""
```
if `new_display` is `None` then you'll get "", and if `new_display` is "" you'll get "" and if it's a string you'll get that string